edwin/client/static/js/components/Timeline.js

```diff
@@ -334,6 +335,25 @@ WhiteboardGroup.propTypes = {
   data: React.PropTypes.object.isRequired,
 };
 
+class NeedinfoGroup extends React.Component {
+  render() {
+    return (
+      <span className="NeedinfoGroup">
+        {this.props.data
+          .map(flag => (
+            <span className="NeedinfoGroup__Data">
+              ni? {flag.get('name', '')}
```

Comment:
To expand, the second parameter to `Immutable.Map::get` is the default value, and is optional. If it isn't provided, `undefined` will be returned, which in this case will have the same effect. Also, I think the code you have in the store guarantees there will always be a name property (even if it is `Unknown`).